infrastructure/cli/pkg/providers/aws/aws.go

```diff
@@ -1,104 +1,578 @@
 package aws
 
 import (
+	"cli/pkg/console"
 	"cli/pkg/kube"
 	"context"
 	"fmt"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"math/rand"
+	"os"
+	"path"
+	"text/template"
+	"time"
+
 	"github.com/aws/aws-sdk-go-v2/config"
 	"github.com/aws/aws-sdk-go-v2/service/ec2"
+	ec2Types "github.com/aws/aws-sdk-go-v2/service/ec2/types"
 	"github.com/aws/aws-sdk-go-v2/service/eks"
-	"github.com/aws/aws-sdk-go-v2/service/eks/types"
-	"log"
-	"os"
+	eksTypes "github.com/aws/aws-sdk-go-v2/service/eks/types"
+	"github.com/aws/aws-sdk-go-v2/service/iam"
+	iamTypes "github.com/aws/aws-sdk-go-v2/service/iam/types"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/mitchellh/go-homedir"
 )
 
+const RolePolicyDocument = `{
+	"Version": "2012-10-17",
+	"Statement": [
+		{
+			"Effect": "Allow",
+			"Principal": {"Service": "eks.amazonaws.com"},
+			"Action": "sts:AssumeRole"
+		},
+		{
+			"Effect": "Allow",
+			"Principal": {"Service": "ec2.amazonaws.com"},
+			"Action": "sts:AssumeRole"
+		}
+	]
+}`
+
+var letters = []rune("abcdefghijklmnopqrstuvwxyz")
+
 type Aws struct {
+	context kube.KubeCtx
 }
 
 func (a *Aws) GetHelmOverrides() []string {
-	return []string{}
+	return []string{"--set", "global.ngrokEnabled=false", "--set", "global.host=http://airy.core"}
```

Comment:
I wouldn't set any global host if it is incorrect. I think instead we should make the host optional on the helm chart because we need to accommodate for the weakest provider constraint.

Reply:
We do not know the host upfront and the helm chart requires this to be set, otherwise will exit with en error.
I think we should set a default for the `global.host` as the chart as a whole should be deplorable with `helm install` without any needed overrides (--values/--set).
For example, the ingress resource doesn't need any specific host to work, but the `hostnames` configMap requires this value, otherwise the apps that use it will crash.
I will do it like that now (put default for `global.host`) and we can open an issue to review this and do it properly in the future.